test/database_service/test_fitval.py

```diff
@@ -29,7 +29,7 @@
 class TestFitVal(QiskitExperimentsTestCase):
     """Test for serialization."""
 
-    __signle_value__ = [
```

Comment:
`signle` to `signal` instead of `single`? Here I am not sure about which was intended, but signal made more sense for me.

Reply:
@nkanazawa1989 I think this was yours?

Reply:
Actually, I think this will be removed as part of the deprecations for 0.4, so you don't have to worry about it.